docs/pages/api-reference/sink_connectors/kafka.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Kafka
+order: 0
+status: published
+---
+### Kafka
+Data sink to any data store that speaks the Kafka protocol (e.g. Native 
+Kafka, MSK, Redpanda etc.)
+
+#### Cluster Parameters
+<Expandable title="name" type="str">
+A name to identify the sink. This name should be unique across ALL connectors.
+</Expandable>
+
+<Expandable title="bootstrap_servers" type="str">
+This is a list of the addresses of the Kafka brokers in a "bootstrap" Kafka 
+cluster that a Kafka client connects to initially to bootstrap itself and discover
+the rest of the brokers in the cluster.
+
+Addresses are written as host & port pairs and can be specified either as a 
+single server (e.g. `localhost:9092`) or a comma separated list of several 
+servers (e.g. `localhost:9092,another.host:9092`).
+</Expandable>
+
+<Expandable title="security_protocol" type='"PLAINTEXT" | "SASL_PLAINTEXT" | "SASL_SSL"'>
+Protocol used to communicate with the brokers. 
+</Expandable>
+
+<Expandable title="sasl_mechanism" type='"PLAIN" | "SCRAM-SHA-256" | "SCRAM-SHA-512" | "GSSAPI"'>
+SASL mechanism to use for authentication. 
+</Expandable>
+
+<Expandable title="sasl_plain_username" type="Optional[str]">
```

Comment:
Can this be given as secret that Hemanth built? If so, maybe update the type signature?

Reply:
@hemanthk269 would be updating the documentation with Secrets entirely soon